docs/software/teamracing.md

```diff
@@ -0,0 +1,45 @@
+---
+template: main.html
+description: TeamRacing allows selection between multiple connected models, failsafing all unselected models.
+---
+
+<img src="https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/software.png">
+
+## What is Team Racing?
+
+Team Racing is a recveiver feature that allows multiple models to be connected to one transmitter at the same time. Each model is assigned to a different switch position and will select that model to control. Any non-selected model is failsafed and will have its telemetry disabled.
```

Comment:
typo: receiver

Reply:
You're the best, thanks for looking it over!